index.js

```diff
@@ -1,4 +1,5 @@
 const Koa = require('koa');
+require('dotenv-safe').config();
```

Comment:
It looks like a config related dependency so we can move it to [`./config.js`](./config.js).